en/references/exercise-description/index.md

```diff
@@ -8,7 +8,11 @@ order: 1
 
 > Dodona supports exercise descriptions and reading activities in HTML and Markdown with several additional features such as support for mathematical formulas.
 
-These features are available for both exercises and reading activities. They are rendered in exactly the same way. The only difference is that exercises have a text editor at the bottom of the exercise description, where a reading activity has a `MARK AS READ` button.
+::: tip Kramdown
+Dodona uses [kramdown](https://kramdown.gettalong.org) to render Markdown, which supports a lot of extensions to the standard Markdown. For more explanation on how this formatting works, you can find in [this overview](https://kramdown.gettalong.org/quickref.html).
+:::
```

Comment:
Make it more clear that this tip is only for advanced users. We don't want people that barely know what markdown is, dive into the kramdown docs.